### PR TITLE
STONEBLD-2371 Tests for multi platform controller

### PR DESCRIPTION
**Pull request by stuartwdouglas**

### CODE REVIEW

**Review by psturc — COMMENTED** [review]
@stuartwdouglas wondering, would it make sense to temporarily update [this script](https://github.com/openshift/release/blob/85869b3dc438e8465f082dd7dc39bc009e129a14/ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh#L109) in this PR so it points to [your e2e-tests PR](https://github.com/redhat-appstudio/e2e-tests/pull/1134/files) and run rehearsal job to validate it works as expected?

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-multi-platform-controller-main-multi-platform-controller-e2e

**Comment by stuartwdouglas** [comment]
@psturc any idea what those failures mean?

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-multi-platform-controller-main-multi-platform-controller-e2e

**Comment by psturc** [comment]
> @psturc any idea what those failures mean?

run `make ci-operator-config` to address the error in [ci-operator-config job](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/51236/pull-ci-openshift-release-master-ci-operator-config-metadata/1788030557486059520)

regarding the error in check-gh-automation job - it complains about 'openshift-cherrypick-robot' not having access to the repo. afaik we don't use that bot in our org so I think we can ignore this error

**Comment by psturc** [comment]
@stuartwdouglas otherwise the rehearsal job needs at least [these changes](https://github.com/redhat-appstudio/e2e-tests/pull/1134#pullrequestreview-2036115374) to be addressed in order to be successful

**Comment by psturc** [comment]
@stuartwdouglas  looks like the label with invalid owners file won't disappear until you become a member of openshift org :/
could you request access to the org? guide for how to do that is [here](https://source.redhat.com/groups/public/openshift/openshift_wiki/openshift_onboarding_checklist_for_github)

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-multi-platform-controller-main-multi-platform-controller-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-multi-platform-controller-main-multi-platform-controller-e2e

**Comment by psturc** [comment]
/pj-rehearse pull-ci-redhat-appstudio-multi-platform-controller-main-multi-platform-controller-e2e

File: ci-operator/config/redhat-appstudio/multi-platform-controller/redhat-appstudio-multi-platform-controller-main.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- as: multi-platform-controller-e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-west-2
+    owner: rhtap
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.13"
```
Comment (psturc) [inline ci-operator/config/redhat-appstudio/multi-platform-controller/redhat-appstudio-multi-platform-controller-main.yaml:39]: we've updated all jobs to use 4.14

```suggestion
    version: "4.14"
```

**Review by psturc — CHANGES_REQUESTED** [review]

**Pull request closed without merging**